pom.xml

```diff
@@ -159,12 +159,12 @@
                     <imageManagement>
                         <image>
                             <repository>${dockerHubPublic}/cafapi/opensuse-jre8</repository>
-                            <tag>4.0.0</tag>
-                            <digest>sha256:52dd8f374d93e3d6fcb32f3a9863c2af16ecd06d0952307e5bf1c7b5530c00d3</digest>
+                            <tag>4.0.1</tag>
+                            <digest>sha256:7fc9a5cb530f51bfcb4c7b1851d80998bb268ea7a3879da85b828c30533674d4</digest>
                         </image>
                         <image>
                             <repository>${dockerHubPublic}/opensuse/leap</repository>
-                            <tag>15.6</tag>
+                            <tag>latest</tag>
```

Comment:
I also updated this as looks like a mistake in https://github.com/CAFapi/opensuse-tomcat-image/pull/97